.travis/publish_docs.sh

```diff
@@ -88,13 +102,21 @@ cp -r target/staging/* "$release_tag"/
 
 printf "${CYAN}Creating redirecting index.html.$RESET \n"
 mv target/build_index.sh "."
-sh ./build_index.sh "$release_tag"
+bash ./build_index.sh "$release_tag"
```

Comment:
travis doesn't load `bash` by default and it runs with the minimal version - `dash`. So, executing other scripts with `sh` was causing problem. So had to execute the scripts exclusively with `bash`